templates/common/on-prem/files/NetworkManager-resolv-prepender.yaml

```diff
@@ -50,15 +50,15 @@ contents:
             show \
             "{{ onPremPlatformAPIServerInternalIP . }}" \
             "{{ onPremPlatformIngressIP . }}")
-        DOMAIN="{{.DNS.Spec.BaseDomain}}"
+        DOMAINS="${IP4_DOMAINS} {{.DNS.Spec.BaseDomain}}"
```

Comment:
While we're at it, should we also do IP6_DOMAINS?

Could also make this reference "set -u" safe by doing ${IP4_DOMAINS:-}, but this script is full of unsafe variable references so I don't know that it actually matters.

Reply:
In regards to `$IP6_DOMAINS`  I've update and repushed.

However, in terms of script saftey, `set -u` would be wise, however:
- we cannot guarantee that NW will set those values
- improve the safety of the script is likely to be disruptive -- the script was not written with safety in mind and turning it on will likely introduce unpredictable behavior changes and is likely an initiative that should be driven by the on-prem platform teams.

Reply:
Yeah, I wouldn't want to enable set -u in this patch.